loops-and-arrays.js

```diff
@@ -53,3 +57,11 @@ const squares = [30, 2, 8, 24, 11];
 //   'Het volume van 24 is 13824',
 //   'Het volume van 11 is 1331'
 // ]
+
+const squares = [30, 2, 8, 24, 11];
+
+console.log(squares);
+for (let i = 0; i < squares.length; i++) {
+    console.log("Het volume van " + squares[i], "is " + (squares[i]= Math.pow(squares[i],3)));
```

Comment:
het is goed dat je method hebt gebruikt. maar toen wij moesten dit zonder method. maar is geen fout.